config/git/hooks/pre-receive.sh

```diff
@@ -0,0 +1,8 @@
+# compares files that match .gitattributes filter to those actually tracked by git-lfs
+diff <(git ls-files ':(attr:filter=lfs)' | sort) <(git lfs ls-files -n | sort) >/dev/null
```

Comment:
I'm able to run this script, but I'm not sure if it's running the expected behavior. 
<img width="1545" alt="Screenshot 2024-10-30 at 5 34 47 PM" src="https://github.com/user-attachments/assets/9be9cbc4-edfa-46b0-bf9b-1e1ba07299d4">